{{cookiecutter.folder_name}}/Makefile

```diff
@@ -44,14 +44,14 @@ tarfiles=$(figures) $(tables) $(styles) $(bibs) $(source)
 all: export flag = ${draft}
 all: main
 
-copy: 
+copy:
```

Comment:
It looks like you are (auto?) removing trailing whitespace. I'm ok with this if it is our explicit convention, but right now it makes the diffs a bit annoying.